deploy.sh

```diff
@@ -10,7 +10,7 @@ blue_bold='\033[1;34m'
 magenta='\033[0;35m'
 reset='\033[0m'     # reset style
 
-deploy_command="serverless deploy $@ --verbose"
+deploy_command="npx serverless deploy $@ --verbose"
```

Comment:
Since `npx` is not always installed on systems that have `npm` installed, and because we don't want to add unnecessary dependencies, we need to first check whether `npx` is available. If it is, then we can prefix the `serverless` command by `npx`, otherwise we'll have to check the version of Serverless to check whether it's properly installed.

As mentioned in a previous comment, in case Serverless is not installed, an error message can be printed that says:

> Serverless is not installed. Please run npm install --save-dev first and try again.

and we can exit from the script after this.